imports/ui/explorer-components/item-list.jsx

```diff
@@ -23,25 +23,25 @@ const updateCounter = (iconType, count) => {
   )
 }
 
-export function CaseList ({ allCases, onItemClick }) {
+export function ItemList ({ allItems, onItemClick, name }) {
   return (
     <div>
-      {allCases.map((caseItem, ind) =>
-        <li key={caseItem.id} className='h2-5 bt b--black-10'>
+      {allItems.map((item, ind) =>
+        <li key={item.id} className='h2-5 bt b--black-10'>
           <div className='flex items-center'>
             <Link
               className={
                 'link flex-grow ellipsis ml3 pl1 ' +
-                  (caseItem.unreadCounts ? 'b ' : '') +
-                  (isClosed(caseItem) ? 'silver strike' : 'bondi-blue')
+                  (item.unreadCounts ? 'b ' : '') +
+                  (isClosed(item) ? 'silver strike' : 'bondi-blue')
               }
-              to={`/case/${caseItem.id}`}
+              to={name === 'report' ? (`report/${item.id}/review`) : (`/case/${item.id}`)}
               onClick={onItemClick}
             >
-              {caseItem.title}
+              {item.title}
             </Link>
-            {updateCounter('announcement', caseItem.unreadCounts && caseItem.unreadCounts.messages)}
-            {updateCounter('assignment_late', caseItem.unreadCounts && caseItem.unreadCounts.updates)}
+            {updateCounter('announcement', item.unreadCounts && item.unreadCounts.messages)}
```

Comment:
This only applies to items of type 'case' so there might be a better way include this into the component